src/library_browser.js

```diff
@@ -861,12 +861,27 @@ var LibraryBrowser = {
           onload();
         }
       }
-    };
-    script.onerror = () => {
+    }
+
+    var loadError = () => {
       {{{ runtimeKeepalivePop() }}}
       if (onerror) onerror();
     };
-    script.src = UTF8ToString(url);
+
+#if ENVIRONMENT_MAY_BE_NODE && DYNAMIC_EXECUTION
+    if (ENVIRONMENT_IS_NODE) {
+      readAsync(url, (data) => {
+        eval(data);
+        loadDone();
+      }, loadError, false);
+      return;
+    }
+#endif
+
+    var script = document.createElement('script');
+    script.onload = loadDone;
+    script.onerror = loadError;
+    script.src = url;
     document.body.appendChild(script);
```

Comment:
I wonder if there could be a function that abstracts over both of these code paths, but if this is the only place we have them then maybe it's fine as it is.